src/js/views/eva-variant-view.js

```diff
@@ -303,8 +303,31 @@ EvaVariantView.prototype = {
         }
     },
 
+    mapEventsFromAccessioningServiceToEventsInfo: function(response){
+        _this = this;
+        var nullOrUndefinedToEmptyElseAddRS = function(value) {
```

Comment:
You can just say getRSFromWSResponse. The implementation details need not be laid out in the function name.

Reply:
updated